spyder_terminal/confpage.py

```diff
@@ -45,7 +45,7 @@ def setup_page(self):
         valid_shells = zip(valid_shells, valid_shells)
         default_option = 'cmd' if WINDOWS else 'bash'
         shell_combo = self.create_combobox(_("Select the shell interpreter:"),
-                                           valid_shells, 'shell', restart=True,
+                                           valid_shells, 'shell',
```

Comment:
Why did you remove `restart=True` from here? I mean, it seems this option requires a restart because I'm not getting a new shell after changing this option in our Preferences.

Reply:
I thought this was another option, sorry hahaha